recipe/meta.yaml

```diff
@@ -1,33 +1,41 @@
 {% set name = "waitress" %}
-{% set version = "2.0.0" %}
+{% set version = "3.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 69e1f242c7f80273490d3403c3976f3ac3b26e289856936d1f620ed48f321897
+  sha256: 682aaaf2af0c44ada4abfb70ded36393f0e307f4ab9456a215ce0020baefc31f
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   entry_points:
     - waitress-serve = waitress.runner:run
 
 requirements:
   build:
-    - python >=2.7
+    - python
     - pip
+  host:
+    - python
+    - setuptools
+    - pip
+    - wheel
   run:
-    - python >=2.7
+    - python
 
 test:
+  requires:
+    - pip
   imports:
     - waitress
   commands:
     - waitress-serve --help
+    - pip check
```

Comment:
No unit tests? Looks like the dependencies for it are slim.

Reply:
Most of the time you can bypass tox since much of the time it's just used as a configuration tool to runs pytest tests. Their pytest configuration is located at https://github.com/Pylons/waitress/blob/main/setup.cfg#L63 and their tox config at https://github.com/Pylons/waitress/blob/main/tox.ini#L10 shows the tox command is really just a pytest command.

Reply:
From what I have tested also implementing pytest -v tests, this fails as pytest can not accurately locate the tests file